README.rst

```diff
@@ -401,6 +406,10 @@ Now you can encrypt a file using::
 
     $ sops encrypt --azure-kv https://sops.vault.azure.net/keys/sops-key/some-string test.yaml > test.enc.yaml
 
+or, without the version:
```

Comment:
The double colons are a shortcut that the next following lines are a code-block (and not a block quote):
```suggestion
or, without the version::
```